functions/go/set-image/image_util/image_util.go

```diff
@@ -0,0 +1,80 @@
+package image_util
```

Comment:
Does this (partially) come  from kustomize?

Reply:
Yes, they are practically the same functions as kustomize, since I need to avoid importing kustomize package, I write my own util package

Reply:
I think if it is the same (or almost the same) as another open source lib, we may need to fork it under a third_party directory with a clear README explaining where the code came from. @yuwenma may have better guidance here

Reply:
I don't think in this setimage use case we should need such a util lib that borrow most of the logic from other libraries. 

If we do, it should be forked. as @natasha41575 suggested